### PR TITLE
Automation of the CEPH-9369 - RADOS: Time scheduled scrubbing: Observe cpu and memory usage during scrubbing

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Test Case: CEPH-9369 - RADOS: Time scheduled scrubbing: Observe cpu and memory usage during scrubbing

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
The code is not tested. Please review the logic of the code.

File: tests/rados/test_scrub_cpu_memory_usage.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import datetime
```
Comment (pdhiran) [inline tests/rados/test_scrub_cpu_memory_usage.py:1]: please add docstrings for the module

Comment (SrinivasaBharath) [reply tests/rados/test_scrub_cpu_memory_usage.py:4]: Added the doc string.

**Comment by SrinivasaBharath** [comment]
Test execution is in progress.

**Review by harshkumarRH — CHANGES_REQUESTED** [review]
@SrinivasaBharath please make the small changes so we can merge this

File: tests/rados/test_scrub_cpu_memory_usage.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        rados_obj: Rados object
+
+    Returns : None
+    """
+    scrub_object.set_osd_configuration("osd_scrub_min_interval", 86400)
+    scrub_object.set_osd_configuration("osd_scrub_max_interval", 604800)
+    scrub_object.set_osd_configuration("osd_scrub_begin_week_day", 0)
+    scrub_object.set_osd_configuration("osd_scrub_end_week_day", 0)
+    scrub_object.set_osd_configuration("osd_scrub_begin_hour", 0)
+    scrub_object.set_osd_configuration("osd_scrub_end_hour", 0)
```
[the remark this answers is not in the document]
Comment (SrinivasaBharath) [reply tests/rados/test_scrub_cpu_memory_usage.py]: Configuration are setting default by using the ceph osd rm config

File: tests/rados/test_scrub_cpu_memory_usage.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                    time.sleep(10)
+                # Break after the verification of 30 minutes
+                break
+            log.info("Scrub not started.Wait for 30 seconds")
+            time.sleep(30)
+        if scrub_check_flag:
+            log.error("The scrub not initiated on the cluster")
+            return 1
+    except Exception as er:
+        log.error(f"Exception hit while command execution. {er}")
```
[the remark this answers is not in the document]
Comment (SrinivasaBharath) [reply tests/rados/test_scrub_cpu_memory_usage.py:127]: Added the return statement.

**Comment by SrinivasaBharath** [comment]
Reef Pass Log- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-FRX86B

**Review by pdhiran — COMMENTED** [review]
looks good

**Review by harshkumarRH — APPROVED** [review]
LGTM

**Review by pdhiran — APPROVED** [review]

**Pull request merged**